src/com/facebook/buck/android/aapt/RDotTxtEntry.java

```diff
@@ -35,7 +35,7 @@
 public class RDotTxtEntry implements Comparable<RDotTxtEntry> {
 
   // Taken from http://developer.android.com/reference/android/R.html
-  public static enum RType {
+  public enum RType {
```

Comment:
removed as Intellij was showing a suggestion that nested enums are implicitly static - http://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.9